blocksync/config.go

```diff
@@ -22,6 +23,7 @@ type Config struct {
 // DefaultConfig is the default config
 var DefaultConfig = Config{
 	Interval:              30 * time.Second,
+	RateLimitInterval:     1 * time.Second,
```

Comment:
can you do some testing to see the sync speed? for 1 sec and 2 sec

Reply:
1sec:  29.6 blks/sec
2sec: 14.4 blks/sec